### PR TITLE
Add husky, lint-staged and prettier standard (closes #51)

### LINKED ISSUE #51: Project linting using ESLint and airbnb guide
https://www.npmjs.com/package/eslint-config-airbnb

### PR DESCRIPTION
**Pull request by morajabi**

### CODE REVIEW

**Comment by ritz078** [comment]
Can you review #53 so that I can merge it before this? Else there will be a lot of conflicts.

File: package.json [hunk 1]
```diff
@@ -5,13 +5,18 @@
...
     "analyze": "cross-env ANALYZE=1 next build",
-    "format": "prettier --write **/*.js",
+    "format": "prettier-standard **/*.js",
     "deploy": "yarn build && cd transform && now",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "**/*.{js,json,css}": ["prettier-standard", "git add"]
```
Comment (ritz078) [inline package.json]: Does lint-stages automatically exclude node_modules ?

File: package.json [hunk 2]
```diff
@@ -67,7 +72,10 @@
     "css-loader": "^0.28.7",
     "deep-diff": "^0.3.8",
     "glob": "^7.1.2",
+    "husky": "^0.14.3",
+    "lint-staged": "^4.1.3",
     "prettier": "^1.6.1",
```
Comment (ritz078) [inline package.json]: I don't think we need prettier anymore.

File: package.json (hunk 1 above)
Comment (morajabi) [reply package.json]: It only executes on staged files in the git, so when we have excluded node_modules in `.gitignore`, node_modules files won't ever hit the staged area :)

File: package.json (hunk 2 above)
Comment (morajabi) [reply package.json]: Yes.

**Pull request merged**